src/app.js

```diff
@@ -10,135 +10,125 @@ import debug from 'debug';
 // Debug log
 const log = debug('watsonwork-helloworld-app');
 
-// Return the list of spaces the app belongs to
-const spaces = (tok, cb) => {
-  request.post('https://api.watsonwork.ibm.com/graphql', {
-    headers: {
-      jwt: tok,
-      'Content-Type': 'application/graphql'
-    },
-
-    // This is a GraphQL query, used to retrieve the list of spaces
-    // visible to the app (given the app OAuth token)
-    body: `
-      query {
-        spaces (first: 50) {
-          items {
-            title
-            id
+// Return a list of spaces the app belongs to
+const spaces = (tok) => {
+  return new Promise((resolve, reject) => {
```

Comment:
I'd suggest to convert this to use the async keyword (vs explicitly returning a Promise) and convert the asynchronous calls inside this function to use await (as this pull request seems to be about converting the sample async/await), instead of having a mix of async/await and promise programming models.